cpp/src/barretenberg/benchmark/honk_bench/compare_honk_branch_vs_baseline.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# This script is used to compare the honk benchmarks between baseline (default: master) and
+# the branch from which the script is run. Simply check out the branch of interest, ensure 
+# it is up to date with local master, and run the script.
+
+# Specify the benchmark suite and the "baseline" branch against which to compare
+BENCH_TARGET="ultra_honk_bench"
+BASELINE_BRANCH="master"
+
+echo -e "\nComparing $BENCH_TARGET between $BASELINE_BRANCH and current branch:"
+# Set some directories
+BASE_DIR="$HOME/barretenberg/cpp"
+BUILD_DIR="$BASE_DIR/build-bench" # matches build dir specified in bench preset
+BENCH_RESULTS_DIR="$BASE_DIR/tmp_bench_results"
+BENCH_TOOLS_DIR="$BUILD_DIR/_deps/benchmark-src/tools"
+
+# Install requirements (numpy + scipy) for comparison script if necessary.
+# Note: By default, installation will occur in $HOME/.local/bin.
+pip3 install -r $BUILD_DIR/_deps/benchmark-src/requirements.txt
```

Comment:
I'd add --user so that it is not a system-wide installation

Reply:
done